### PR TITLE
Add better XDomain support

### PR DESCRIPTION
**Pull request by westeri**

### CODE REVIEW

File: tbtacl/CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 set(TBTACL "tbtacl")
+set(TBTXDOMAIN "tbtxdomain")
```
Comment (ybernat) [inline tbtacl/CMakeLists.txt]: Isn't it better to keep it at a separated dir?

File: tbtadm/controller.cpp [hunk 2]
```diff
@@ -308,6 +319,30 @@ void tbtadm::Controller::devices()
...
+            continue;
+        }
+
+        chdir(dir.path());
+
+        // TODO: better formatting
+        const auto routeString = dir.path().filename().string();
+
+        m_out << routeString << '\t' << readVendor(vendorFilename) << '\t'
+              << readDevice(deviceFilename) << std::endl;
```
Comment (ybernat) [inline tbtadm/controller.cpp:342]: Maybe we want more details here? E.g. the services it supports? (But this can be added later if decided.)

**Review by ybernat — COMMENTED** [review]
Just 2 questions, but apart from them it looks excellent!
Thanks!

File: tbtacl/CMakeLists.txt (hunk 1 above)
Comment (westeri) [reply tbtacl/CMakeLists.txt]: Sure, I can move it to separate dir. How about udev rules, should I make a separate rule that then gets installed as 60-tbtxdomain.rules or do you think it is better to merge them?

File: tbtadm/controller.cpp (hunk 2 above)
Comment (westeri) [reply tbtadm/controller.cpp:342]: Yup, I think we can add those details later as needed :)

File: tbtacl/CMakeLists.txt (hunk 1 above)
Comment (ybernat) [reply tbtacl/CMakeLists.txt]: If we prefer to separate the scripts to separated dirs, the udev rules should be separated too.
But maybe the udev rules are a reason to keep them together?

Comment (westeri) [reply tbtacl/CMakeLists.txt]: Right that was the reason I put it originally to the same directory. However, I think it makes more sense to have them in a separate directories and have separate udev rules. I'll update the patch accordingly.

**Comment by westeri** [comment]
The updated pull request now moves tbtxdomain to a separate directory.

**Review by ybernat — APPROVED** [review]
Thanks again!

**Pull request merged**